### DISCUSSION TITLE
Logs have grown to 68 files. Consider checking them out and deleting them.

### DISCUSSION THREAD

**Discussion by neoey** [Rem0o/FanControl.Releases#2408] [Q&A]
When I start my PC i always get this notification.
"Logs have grown to 68 files. Consider checking them out and deleting them."
I have opened the error log, but it always shows me only the last one or two. Where can i find all of them and delete them or disable the popup?

**Comment by Rem0o** [comment]
They are located in the FanControl folder, basically where you extracted it or installed it.  If it has grown to 68 files long, maybe you got an issue? Care to share what error is repeated over and over?

**Reply by neoey** [reply]
![image](https://github.com/Rem0o/FanControl.Releases/assets/89702594/96bae365-eb7c-46ca-93ab-d9829393bc48)
Is this the right folder? I cant find them here. I might have an issue, but idk what it would be or how it happens.

**Reply by Rem0o** [reply]
They should be in there.... might you have multiple installations?
